presto-main/src/main/java/com/facebook/presto/sql/planner/BasePlanFragmenter.java

```diff
@@ -383,240 +394,6 @@ private PlanNode createRemoteMaterializedExchange(ExchangeNode exchange, Rewrite
         return visitTableScan(scan, context);
     }
 
-    private PartitioningVariableAssignments assignPartitioningVariables(Partitioning partitioning)
```

Comment:
Moved to https://github.com/prestodb/presto/pull/20887/files#diff-6b68e5660ffcbb5b4436b88cf5c58be0a90a1117a0b42d1eec80cf1d076f08e6

Reply:
Why move it?

Reply:
By utilizing similar code in PhysicalCteOptimizer, we can reduce redundancy through consolidation. Moreover, these utility functions are unrelated to the fragmenter